bins/packages/rfs/rfs.sh

```diff
@@ -1,22 +1,31 @@
-RFS_VERSION="2.0.0"
-RFS_CHECKSUM="d6e1c3036d8b0d7d6df4d67fe17a0502"
-RFS_LINK="https://github.com/threefoldtech/rfs/releases/download/v${RFS_VERSION}/rfs"
+RFS_VERSION_V1="1.1.1"
+RFS_VERSION_V2="2.0.0"
+RFS_CHECKSUM_V1="974b8dc45ae9c1b00238a79b0f4fc9de"
+RFS_CHECKSUM_V2="d6e1c3036d8b0d7d6df4d67fe17a0502"
+RFS_LINK_V1="https://github.com/threefoldtech/rfs/releases/download/v${RFS_VERSION_V1}/rfs"
+RFS_LINK_V2="https://github.com/threefoldtech/rfs/releases/download/v${RFS_VERSION_V2}/rfs"
 
 download_rfs() {
-    download_file ${RFS_LINK} ${RFS_CHECKSUM} rfs-${RFS_VERSION}
+    download_file ${RFS_LINK_V1} ${RFS_CHECKSUM_V1} rfs-${RFS_VERSION_V1}
+    download_file ${RFS_LINK_V2} ${RFS_CHECKSUM_V2} rfs-${RFS_VERSION_V2}
 }
 
 prepare_rfs() {
     echo "[+] prepare rfs"
-    github_name "rfs-${RFS_VERSION}"
+    github_name "rfs-${RFS_VERSION_V1}"
+    github_name "0-fs-${RFS_VERSION_V2}"
```

Comment:
You can only produce one github_name. That will be `rfs-${RFS_VERSION_V2}`

basically that is gonna be the name and version of the produced pacakge.
this one will have then 2 binaries, rfs and g8ufs